samples/.net/endpoints/batch/Clients/AuthenticationClient.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="AuthenticationClient.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+using Azure.Core;
+
+namespace BatchInferencingSamples
+{
+    internal class AuthenticationClient
+    {
+        /// <summary>
+        /// Gets an access token to score.
+        /// </summary>
+        /// <param name="credentials">The user's credentials.</param>
+        /// <returns>A string representation of the access token.</returns>
+        public async Task<string> GetAccessToken(TokenCredential credentials)
+        {
+            // Given this is a dataplane operation we need the authentication scope to come from Azure Machine Learning
+            string[] scopes = new string[]
+            {
+                "https://ml.azure.com/.default"
```

Comment:
What is the default? Isn't the scope just ml.azure.com?

Reply:
It's the recommended way in c#!